src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exeptions.InvalidUserDataException;
+import core.basesyntax.exeptions.NullUserException;
```

Comment:
There is a typo in the package name `exeptions`. It should be `exceptions` to maintain consistency and adhere to standard naming conventions.